cortex-cpp/main.cc

```diff
@@ -18,6 +20,27 @@
 #endif
 
 int main(int argc, char* argv[]) {
+  // Check if this process is for python execution
+  if (argc > 1) {
+    if (strcmp(argv[1], "--run_python_file") == 0) {
+      std::string py_home_path = (argc > 3) ? argv[3] : "";
+      std::unique_ptr<cortex_cpp::dylib> dl;
+      try {
+        std::string abs_path = cortex_utils::GetCurrentPath() +
+                               cortex_utils::kPythonRuntimeLibPath;
+        dl = std::make_unique<cortex_cpp::dylib>(abs_path, "engine");
+      } catch (const cortex_cpp::dylib::load_error& e) {
+        LOG_ERROR << "Could not load engine: " << e.what();
+        return 1;
+      }
+
+      auto func = dl->get_function<CortexPythonEngineI*()>("get_engine");
+      auto e = func();
+      e->ExecutePythonFile(argv[0], argv[2], py_home_path);
+      return 0;
+    }
+  }
+
```

Comment:
As I remember it's a test script, we can remove it out in prod

Reply:
No, it's not. In reality, we create another process to run the fine-tune job with parameters include `python-file` (which is fine-tune script)

Reply:
I didn't see the location; this is actually the main entry of the engine, not the program.